src/main/java/seedu/duke/model/item/Link.java

```diff
@@ -2,6 +2,10 @@
 
 package seedu.duke.model.item;
 
+import seedu.duke.DukeException;
+
```

Comment:
can omit out the whitespace

Reply:
This is fine, IntelliJ splits up normal imports from other classes, exception imports and Java classes imports.

Example:
![image](https://user-images.githubusercontent.com/1352844/98431535-2a129480-20f1-11eb-9124-91189925b5b7.png)

Reply:
Ahh alright then